### ISSUE TITLE
Placeholder to show name of Player's job 

### ISSUE THREAD

**Issue by Seeroh** [Zrips/Jobs#524]
**Description of new idea or improvement on existing one**
Currently, there is a placeholder to display the levels and EXP of each job for players, but no placeholder to display the NAME of the job, to correspond.

**Detailed information what should do what and when**
For instance, I can use `%jobsr_user_jlevel_2%` to show the level for the player's second job. Lets say it's Builder. But there is no placeholder to show the Name for player's second job. If I used `%jobsr_name_2%`, it would show the second job in the config, which might be Farmer.

So I can't have a menu, scoreboard, or gui, that shows the player's second job, with the level of the job next to it.

It'd be great if there were a placeholder that did this.

So basically, something like `%jobsr_user_name_(jname/number)%`.

**Comment by montlikadani** [comment]
So now you want to show the job name or the user name in the job?

**Comment by Seeroh** [comment]
I'd like to show the name of the job in X slot, for the player.

So if the player has three jobs: Farmer, Builder, Digger, I could use a placeholder like %jobsr_jname_2% to return "Builder", or %jobsr_jname_3% to return "Digger".

**Comment by montlikadani** [comment]
Added in [this](https://github.com/Zrips/Jobs/commit/cfc0453cfcf79f438f402c6e3fd3044875fd6bee) commit.
